src/screens/auth/containers/formLogin.js

```diff
@@ -0,0 +1,116 @@
+import {View, Text, StyleSheet, TouchableOpacity} from "react-native"
+import React, {useState} from "react"
+import {Input} from "react-native-elements"
+import Color from "@common/Color"
+import ButtonLogin from "@components/button"
+import SocialMethod from "./socialMethod"
+import EyeIcon from "./icon/eyeIcon"
+import EyeOffIcon from "./icon/eyeOffIcon"
+
+const FormLogin = () => {
+  const [isFocus, setIsFocus] = useState({acount: false, password: false})
+  const [isVisible, setIsVisible] = useState(false)
+  const [account, setAccount] = useState("")
+  const [password, setPassword] = useState("")
+
+  const handleInputFocus = (text) => {
+    setIsFocus({
+      [text]: true,
+    })
+  }
+  const handleInputBlur = (text) => {
+    setIsFocus({
+      [text]: true,
+    })
+  }
+
+  const showPasswords = () => {
+    setIsVisible(!isVisible)
+  }
+
+  const onChangeAccount = (value) => {
+    setAccount(value)
+  }
+
+  const onChangePassword = (value) => {
+    setPassword(value)
+  }
+
+  return (
+    <View style={styles.container}>
+      <Input
+        autoFocus
+        placeholder="Nhập số điện thoại của bạn"
+        inputContainerStyle={styles.inputStyle}
+        value={account}
+        onChangeText={onChangeAccount}
+        containerStyle={
+          !isFocus.acount
```

Comment:
Avoid negative conditional.
```suggestion
          isFocus.acount
           ? styles.inputContainerHover
           : styles.inputContainerStyle
```